### DISCUSSION TITLE
Build errors in Vercel due to typescript

### DISCUSSION THREAD

**Discussion by SairajPrince** [vercel/next.js#40538] [Help]
Can you please specify the correct versions of react, react-dom and types/react, types/react-dom to be used in NextJs sitecore JSS project, we are facing issues currently with Vercel build.
![image](https://user-images.githubusercontent.com/7039898/190048216-708d2e4f-2370-4f98-b27c-5ed6a163e3dc.png)

**Comment by anshuman-8** [comment]
I have recently deployed a site and these were the dependencies and devDependencies 

 "dependencies": {
    "next": "12.3.0",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-icons": "^4.4.0"
  },
  "devDependencies": {
    "@types/node": "18.7.17",
    "@types/react": "18.0.19",
    "@types/react-dom": "18.0.6",
    "autoprefixer": "^10.4.9",
    "eslint": "8.23.1",
    "eslint-config-next": "12.3.0",
    "postcss": "^8.4.16",
    "tailwindcss": "^3.1.8",
    "typescript": "4.8.3"
  }

**Reply by SairajPrince** [reply]
Need this versions specific to Sitecore JSS NextJS project, please help.
